File: Calico/All Star Code/Mash.py
#Game adapted from the game MASH#

from random import *

print ("Let's play MASH")

myWives=["Justin Bieber","Miley Cyrus","Kim Kardasian"]
myWives.append(raw_input("Who do you want to marry?"))


myPets=["Donkey","Cow","Rat"]
myPets.append(raw_input("What type of pet do you want?"))

myRec=["Striper","Con-artist","Terrorist"]
myRec.append(raw_input("What do you want to be known as?"))

myCars=["Mercedes-Benz Maybach Exelero","Lexus RX 350", "Ford Escape"]
myCars.append(raw_input("What type of car do you want?"))

print ("You'll marry...")
print (choice(myWives))

print ("Your pet will be a...")
print (choice(myPets))

print ("You will be known as a...")
print (choice(myRec))

print ("Your car will be a...")
print (choice(myCars))

File: JavaScript/Mash.js
//Game adapted from the game MASH#

alert ("Let's play MASH");

var myWives=["Justin Bieber.","Miley Cyrus.","Kim Kardasian."];
var wiveInput=prompt("Who do you want to marry?");
myWives.push(wiveInput+".");
var Wives=myWives[Math.floor(Math.random()*myWives.length)];

var myPets=["donkey.","cow.","rat."];
var petInput=prompt("What type of pet do you want?");
myPets.push(petInput+".");
var Pets=myPets[Math.floor(Math.random()*myPets.length)];

var myRec=["striper.","con-artist.","terrorist."];
var recInput=prompt("What do you want to be known as?");
myRec.push(recInput+".");
var Rec=myRec[Math.floor(Math.random()*myRec.length)];

var myCars=["Mercedes-Benz Maybach Exelero.","Lexus RX 350.", "Ford Escape."];
var carInput=prompt("What type of car do you want?");
myCars.push(carInput+".");
var Cars=myCars[Math.floor(Math.random()*myCars.length)];


alert ("You'll marry "+Wives);

alert ("Your pet will be a "+Pets);

alert ("You will be known as a "+Rec);

alert ("Your car will be a "+Cars);
